@here/harp-mapview/lib/text/TextStyleCache.ts

```diff
@@ -35,7 +35,7 @@ export function computeStyleCacheId(
     technique: Technique,
     zoomLevel: number
 ): string {
-    return `${datasourceName}_${technique.renderOrder}_${zoomLevel}`;
+    return `${datasourceName}_${(technique as any)._index}_${zoomLevel}`;
```

Comment:
This is hard issue (interpolated text or POI properties1!), so it's still workaround, but ..

Why not `Math.floor(zoomLevel)` here, so patch is smaller ?

Reply:
Interpolated text and poi properties need a more complex setup (since their runtime evaluation is way different from all the other objects, look at ``TextElementsRenderer.renderTextElements()``), so until we don't rework the text placement system, we will not allow interpolation of these properties.

And the patch is minimal, and looks more correct to explicitly floor the input to this function to do it behind the curtains (inside the function body).